src/synthesizer/parametric/morphology.py

```diff
@@ -385,3 +385,64 @@ def compute_density_grid_from_arrays(self, xx, yy, units=kpc):
                 "Only kpc and milliarcsecond (mas) units are supported "
                 "for morphologies."
             )
+
+
+class Gaussian2D(MorphologyBase):
+    def __init__(
+        self, x=None, y=None, x_mean=0, y_mean=0, stddev_x=1, stddev_y=1, rho=0
+    ):
+        # Initialise obj with params:
+        self.x = x
+        self.y = y
+        self.x_mean = x_mean
+        self.y_mean = y_mean
+        self.stddev_x = stddev_x
+        self.stddev_y = stddev_y
+        self.rho = rho
+
+    # define 2D Gaussian matrix
+    def gaussian2D_mat(self):
```

Comment:
This function needs to take the `x` and `y` variables rather than attaching them to the `Gaussian2D` instance.